pkg/ui/cluster-ui/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@npm_cluster_ui//webpack-cli:index.bzl", webpack = "webpack_cli")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("@npm//@bazel/typescript:index.bzl", "ts_project")
+
+ts_project(
+    name = "cluster_ui_ts_project",
+    srcs = glob(["src/**"]),
+    tsconfig = "tsconfig.json",
+    emit_declaration_only = True,
+    declaration = True,
+    deps = [
+        "@npm_cluster_ui//@babel/parser",
+        "@npm_cluster_ui//@types",
+        "@npm_cluster_ui//@babel/types",
+      #  "//pkg/ui/src/js:ui_protos_lib",
+      #  "@npm_cluster_ui//@cockroachlabs/crdb-protobuf-client",
+      # "@npm_cluster_ui//:node_modules",
```

Comment:
`cluster_ui_ts_project` this is a target which I want to build.
when I uncomment these deps then build fails without any descriptive information, otherwise it fails because missing typescript typings